n3fit/src/n3fit/model_gen.py

```diff
@@ -637,7 +639,7 @@ def layer_fitbasis(x):
 
         # Rotation layer, changes from the 8-basis to the 14-basis
```

Comment:
Please update comments such as these, or better, just add the photon layer in a differen line for the sake of readability. Even if this is not the final version, it's likely to end up in the final version if done sloppy initially